src/Data/TreeDiff/Golden.hs

```diff
@@ -52,8 +56,6 @@ ediffGolden impl testName fp x = impl testName expect actual cmp wrt
     cmp a b
         | a == b    = return Nothing
         | otherwise = return $ Just $
-            setSGRCode [Reset] ++ showWL (ansiWlEditExprCompact $ ediff a b)
-    wrt expr = BS.writeFile fp $ TE.encodeUtf8 $ T.pack $ showWL (WL.plain (ansiWlExpr expr)) ++ "\n"
-
-showWL :: WL.Doc -> String
-showWL doc = WL.displayS (WL.renderSmart 0.4 80 doc) ""
```

Comment:
why this change. AFAICT (by using this package very day), `ediffGolden` colors output?

Reply:
Correct, there's no change in behavior here. There's a small change in the implementation because the `prettyprint` libraries have a different interface than the `ansi-wl-pprint` compatibility layer.